### PR TITLE
Remove survey / data features. Resolves GH-1462

### PR DESCRIPTION
**Pull request by thisandagain**

### CODE REVIEW

**Comment by thisandagain** [comment]
@Pomax R?

File: views/add/index.js [hunk 1]
```diff
@@ -29,9 +29,11 @@ module.exports = view.extend({
     ],
     inputBlocks: [
       // Form / data blocks
+      /*
```
Comment (Pomax) [inline views/add/index.js]: let's just remove these, they'll live on in the commit history

File: views/detail/index.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
             <div class="app-icon-image" style="background-image: url('{{app.iconImage}}')"></div>
         </div>
 
-        <div v-show="isAdmin" v-component="switch" v-with="options: ['My App', 'App Data']"></div>
+        <!-- <div v-show="isAdmin" v-component="switch" v-with="options: ['My App', 'App Data']"></div> -->
```
Comment (Pomax) [inline views/detail/index.html]: same here. rather than comment, easier to delete entirely

**Comment by Pomax** [comment]
nice PR, although I'm always in favour of "if we're removing things, the history will keep them for us to restore at a later date"

**Comment by thisandagain** [comment]
:+1: Will remove.

**Comment by thisandagain** [comment]
All set. @Pomax re-R?

**Comment by Pomax** [comment]
land it!

**Pull request merged**